arc/modal/public_api.py

```
@@ -43,6 +48,9 @@
             # Default URL pattern for Modal web endpoints
             workspace = os.environ.get("ARC_MODAL_WORKSPACE", "your-workspace")
             base_url = f"https://{workspace}--arc-production-evaluate-scenario-endpoint.modal.run"
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)